gems/pending/VMwareWebService/MiqVimInventory.rb

```diff
@@ -1833,7 +1833,7 @@ def storagePods_locked
     ensure
       @cacheLock.sync_unlock if unlock
     end
-    $vim_log.info "MiqVimInventory.dataStores_locked: loaded Datastore cache for #{@connId}"
+    $vim_log.info "MiqVimInventory.storagePods_locked: loaded Datastore cache for #{@connId}"
```

Comment:
Instead of `MiqVimInventory.storagePods_locked`, you can do `"#{self.class.name}##{__method__}: ....."`